material/economics/bitcoin-is-not-money.toml

```diff
@@ -1,4 +1,4 @@
-tags="money"
+tags="money, bitcoinstandard"
```

Comment:
It might be better to use an inline array `["money", "bitcoinstandard"]` instead of a comma-separated string. This way the TOML parser already converts it to an array.

Reply:
I had the same thought but decided to use a simple string instead to ease contribution, exceptions are links as an array broken in multiple lines is more readable.